src/components/Card/__tests__/__snapshots__/Card.test.tsx.snap

```diff
@@ -430,6 +430,7 @@ exports[`Card shows Card with non-default elevation 1`] = `
 
 exports[`Card shows Card with ripple feedback and no onClick 1`] = `
 .c0 {
+  cursor: pointer;
```

Comment:
Card.tsx line 42 checks if onClick === defaultOnClick and sets the cursor to pointer - we need to update it to check if it's the same as whatever the new default is so it doesn't make the cursor a pointer for all cards

Reply:
I changed it to make the comparison in the Card component and passed the result as a prop to the container